src/components/Checkbox/__stories__/Checked.tsx

```diff
@@ -0,0 +1,31 @@
+import Checkbox from '..'
+
+const codeSnippet = `
+<Checkbox checked onChange={() => {}}>
+  Checked checkbox
+</Checkbox>
+<Checkbox checked="indeterminate" onChange={() => {}}>
+  Indeterminate checkbox
+</Checkbox>
+`
+export const Checked = () => (
+  <>
+    <Checkbox checked onChange={() => {}}>
+      Checked checkbox
+    </Checkbox>
+    <Checkbox checked="indeterminate" onChange={() => {}}>
+      Indeterminate checkbox
+    </Checkbox>
+  </>
+)
+
+Checked.parameters = {
+  docs: {
+    storyDescription:
+      'Checkbox can have two state `checked` or `indeterminate` defined by prop `checked`.',
+    source: {
+      code: codeSnippet,
+      format: true,
+    },
+  },
+}
```

Comment:
This can greatly be simplified with something like this:

```tsx
import { ComponentStory } from '@storybook/react'
import Checkbox from '..'

export const Checked: ComponentStory<typeof Checkbox> = props => (
  <>
    <Checkbox checked onChange={() => {}}>
      Checked checkbox
    </Checkbox>
    <Checkbox checked="indeterminate" onChange={() => {}}>
      Indeterminate checkbox
    </Checkbox>
  </>
)

Checked.parameters = {
  docs: {
    storyDescription:
      'Checkbox can have two state `checked` or `indeterminate` defined by prop `checked`.',
  },
}

```

Not sure how to remove the fragment `<>` from code snippet but I think it's not that big of a deal